### ISSUE TITLE
Contact count includes disabled/inactive addressbooks

### ISSUE THREAD

**Issue by wunschadresse** [nextcloud/contacts#3808]
### Describe the bug

Not necessarily a bug, but as user, I would expect a different behavior. 

### Steps to reproduce

1. Go to contacts
2. have several addressbooks shared with you (i did not test using owned addressbooks)
3. check the count next to your all members/single contact groups
4. disable one adressbook
5. check the count again

### Expected behavior

Count should change, based on the active addressbooks.

### Actual behavior

Count includes inactive address books

### Contact version

5.5.1

### Operating system

Win10 + Firefox 122.0.1 (64-Bit)

### PHP engine version

None

### Web server

None

### Database

None

**Comment by SebastianKrupinski** [comment]
Backports are in, they will be shipped some time next week.

Please let us know if this is still an issue after the update.

**Comment by roccostorm** [comment]
I have run the update and the 

Contact Version: 7.1.1

- Contact Count for Groups is still wrong
- Contact Birthdays from Disabled Addresbooks are still shown in the Calendar

**Comment by MicKress** [comment]
Contact Version 7.11 - same behavior as mentioned before.

**Comment by hamza221** [comment]
Hello, 
I just double checked the fix was released on 7.1.1. 
Can you please remove browser cache and reload the page. and let me know if the bug still persists

**Comment by hamza221** [comment]
Accidental close still waiting for confirmation

**Comment by roccostorm** [comment]
As you can see in this screenshoots, the "All Contacts" counter is updated if an adressbook is disabled, but the counter for the groups is still wrong.
<img width="235" alt="Image" src="https://github.com/user-attachments/assets/5d07789b-5fca-4d6e-9347-ec4cf70f1803" />
<img width="389" alt="Image" src="https://github.com/user-attachments/assets/04e381b2-3129-45e4-80c0-1ce08188d153" />

In addition, I have still two more problems:
- brithdays from contacts of a diasabled adressbook are shown in calendar app
- I have a  adressbook that shows zero contacts since the last update, but when I sync it via caldav, it is not empty.

But this may be seperate problems not related to this ticket.

**Comment by hamza221** [comment]
>As you can see in this screenshoots, the "All Contacts" counter is updated if an adressbook is disabled, but the counter for the groups is still wrong.

Just to make sure, when you refresh the page it gets updated right?

>brithdays from contacts of a diasabled adressbook are shown in calendar app

>I have a adressbook that shows zero contacts since the last update, but when I sync it via caldav, it is not empty.

Can you please open new issues for these

**Comment by roccostorm** [comment]
Yes. The "All Contacts" Counter is updated imediately, the contact groups only after reloading.

**Comment by hamza221** [comment]
I will close this issue as the bug in question is solved. 
I opened a new issue #4536 for the contact group count
